src/api/builds.ts

```diff
@@ -127,16 +125,15 @@ export async function triggerBuild(
  * @export
  * @param {string} buildSid the build to wait for
  * @param {string} serviceSid the service of the build
- * @param {GotClient} client API client
+ * @param {TwilioServerlessApiClient} client API client
  * @param {events.EventEmitter} eventEmitter optional event emitter to communicate current build status
  * @param {number} timeout optional timeout. default: 5 minutes
  * @returns {Promise<void>}
  */
 export function waitForSuccessfulBuild(
   buildSid: string,
   serviceSid: string,
-  client: GotClient,
-  eventEmitter: events.EventEmitter,
```

Comment:
I think it would be nice to keep this as an optional argument. In case people want to have a separate event emitter that deals with build status updates

Reply:
This was weird, because it lead to the call to `waitForSuccessfulBuild(bSid, sSid, this, this)` inside the `TwilioServerlessApiClient`. If we create a standalone HTTPClient that would change again though, so I can switch this back to include the event emitter.